CoreUtilities/src/au/gov/asd/tac/constellation/utilities/svg/SVGData.java

```diff
@@ -247,22 +257,22 @@ public void cleanAttributes() {
      * Recursive function to generate a string equivalent of complex SVG data 
      * captured within this object.
      * @param prefix
-     * @return String representation of the current element and all of it's child elements.
+     * @return an ArrayList<String> representation of the current element and all of it's child elements.
      */
-    private String toString(final String prefix) {
-        final StringBuilder svgString = new StringBuilder();
+    private ArrayList<String> toString(final String prefix) {
+        final ArrayList<String> svgString = new ArrayList<>();
```

Comment:
does this need to be serailised? If not, the return type and declaration here can be List